lib/rules/sort-prop-types.js

```diff
@@ -83,6 +85,78 @@ module.exports = {
         return;
       }
 
+      function fix(fixer) {
+        function sortInSource (allNodes, source) {
+          const originalSource = source;
+          const nodeGroups = allNodes.reduce((acc, curr) => {
+            if (curr.type === 'ExperimentalSpreadProperty' || curr.type === 'SpreadElement') {
+              acc.push([]);
+            } else {
+              acc[acc.length - 1].push(curr);
+            }
+            return acc;
+          }, [[]]);
+
+          nodeGroups.forEach(nodes => {
+            const sortedAttributes = nodes.slice(0).sort((a, b) => {
```

Comment:
This sorting function should probably be defined at file level, since it’s pure?